joeflow/views.py

```diff
@@ -28,6 +28,11 @@ def get_template_names(self):
         )
         return names
 
+class StartWorkflowMixin():
+    """
+    Use this mixin to create a start workflow with a view.
+    """
+    pass
```

Comment:
`pass` isn't needed here, the doc string should be in a single line, or preferably longer / more descriptive.

```suggestion
    """View-mixin to create a start workflow."""
```